core/src/main/java/org/frankframework/http/rest/ApiListenerServlet.java

```diff
@@ -476,11 +475,15 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 						MultipartMessages parts = MultipartUtils.parseMultipart(request.getInputStream(), request.getContentType());
 						for (Entry<String, Message> entry : parts.messages().entrySet()) {
 							String fieldName = entry.getKey();
-							if((body == null && multipartBodyName == null) || fieldName.equalsIgnoreCase(multipartBodyName)) {
+							if (!listener.isParameterAllowed(fieldName)) {
+								LOG.warn("Request contains multipart field [{}] which is not allowed", fieldName);
+								continue;
+							}
```

Comment:
Dit moet ook nog gedocumenteerd worden of mogelijk een andere setter worden...

Reply:
Ik heb een paar extra comments toegevoegd aan de documentatie maar voor multipart names is geloof ik geen directe setter, behalve setMultipartBodyName?